app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/reducer/actions.ts

```diff
@@ -57,7 +57,11 @@ function addBranch(state: FormBranchesState) {
   } else {
     treatmentBranches = [
       ...(treatmentBranches || []),
-      createAnnotatedBranch(state.lastId, `treatment ${lastId}`),
+      createAnnotatedBranch(
+        state.lastId,
+        // 65 is A, but we generate a control, so start at 64
+        `Treatment ${String.fromCharCode(64 + lastId)}`,
```

Comment:
Things will get weird if we ever go over 26 branches.

(but that's a weird number of branches)

![Capture](https://user-images.githubusercontent.com/21687/121974680-c7434480-cd34-11eb-8495-72854caa3d4e.PNG)

Reply:
Ehh, I'd be willing to revisit if we're regularly going past 26 branches. Not super concerned at this point!